README.md

```diff
@@ -418,7 +442,7 @@ TEST=create_event_transactional_stream ./gradlew robotest
 cp -f build/libs/mongoose-storage-driver-pravega-*.jar ~/.mongoose/<MONGOOSE_BASE_VERSION>/ext/
 ```
 Note that the Pravega storage driver depends on the 
-[Coop Storage Driver](http://repo.maven.apache.org/maven2/com/github/emc-mongoose/mongoose-storage-driver-coop/) 
+[Preempt Storage Driver](http://repo.maven.apache.org/maven2/com/github/emc-mongoose/mongoose-storage-driver-preempt/)
```

Comment:
Please consider such change also in the "Deployment -> Basic" section.
Then you can skip ci (if you update the documentation only)

Reply:
This change is on master already. I just noticed that in this section it didn't get updated.